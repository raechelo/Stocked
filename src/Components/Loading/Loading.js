import React from 'react';
import veggie from '../../assets/veggie.gif';

export const Loading = () => {
  return (
    <div>
      <h4>Loading...</h4>
      <img src={veggie} alt='loading gif' />
    </div>
  )
}

export default Loading;