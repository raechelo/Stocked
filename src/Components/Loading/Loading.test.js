import React from 'react';
import Loading from './Loading';
import { shallow } from 'enzyme';

describe('Loading', () => {

  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Loading />)
  });

  it('should match the snapshot with all the data passed in', () => {
    expect(wrapper).toMatchSnapshot();
  });
});