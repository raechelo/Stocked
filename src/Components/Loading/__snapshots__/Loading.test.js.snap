// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should match the snapshot with all the data passed in 1`] = `ShallowWrapper {}`;
